solutions/silver/cf-1879D.mdx

```diff
@@ -87,6 +87,90 @@ int main() {
 ```
 
 </CPPSection>
+
+<JavaSection>
+
+```java
+import java.util.*;
+
+public class SumofXORFunctions {
+	static int MOD = 998244353;
```

Comment:
1. rename class to `SumOfXOR`
2. make the mod final

Reply:
done.